Tribler/Core/Modules/restapi/settings_endpoint.py

```diff
@@ -89,19 +89,7 @@ def parse_setting(self, section, option, value):
         if section == "libtorrent" and (option == "max_download_rate" or option == "max_upload_rate"):
             self.session.lm.ltmgr.update_max_rates_from_config()
 
-        # Apply changes to the default downloadconfig to already existing downloads
-        if section == "download_defaults" and option in ["seeding_mode", "seeding_time", "seeding_ratio"]:
-            for download in self.session.get_downloads():
-                if download.config.get_credit_mining():
-                    # Do not interfere with credit mining downloads
-                    continue
-                elif option == "seeding_mode":
-                    download.config.set_seeding_mode(value)
-                elif option == "seeding_time":
-                    download.config.set_seeding_time(value)
-                elif option == "seeding_ratio":
-                    download.config.set_seeding_ratio(value)
-        elif section == 'credit_mining' and option == 'enabled' and \
+        if section == 'credit_mining' and option == 'enabled' and \
```

Comment:
Shouldn't we be setting the download defaults here? Otherwise, nothing will happen.

Reply:
No, we can leave that out. All settings are automatically copied by
https://github.com/Tribler/tribler/blob/2cc456f4c3f8d4e94cd8acfda8235fedba535ef8/Tribler/Core/Modules/restapi/settings_endpoint.py#L83-L86

I've checked it out, it works as expected.